website/content/docs/secrets/mongodbatlas.mdx

```diff
@@ -2,49 +2,45 @@
 layout: docs
 page_title: MongoDB Atlas - Secrets Engines
 description: |-
-  The MongoDB Atlas Secrets Engine for Vault generates MongoDB Atlas
+  The MongoDB Atlas secrets engine for Vault generates MongoDB Atlas
   Programmatic API Keys dynamically.
 ---
 
 # MongoDB atlas secrets engine
 
-The MongoDB Atlas Secrets Engine generates Programmatic API keys. The created MongoDB Atlas secrets are
+The MongoDB Atlas secrets engine generates Programmatic API keys. The created MongoDB Atlas secrets are
 time-based and are automatically revoked when the Vault lease expires, unless renewed.
 
 Vault will create a Programmatic API key for each lease that provide appropriate access to the defined MongoDB Atlas
 project or organization with appropriate role(s). The MongoDB Atlas Programmatic API Key Public and
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
-[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
+[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/configure-api-access/#programmatic-api-keys).
```

Comment:
BTW this seems to redirect to https://www.mongodb.com/docs/atlas/configure-api-access/#programmatic-api-keys now

Reply:
Thanks! Updated all links to the new domain